src/Nfield.Quota/QuotaFrameValidator.cs

```diff
@@ -225,6 +231,34 @@ public QuotaFrameValidator()
             return !hasInvalidChilds;
         }
 
+        private static bool HaveValidTargets(
+            QuotaFrame frame,
+            ICollection<QuotaFrameVariable> frameVariables,
+            PropertyValidatorContext context)
+        {
+            var inValidTarget = false;
+            if (frame.Target < 0)
+            {
+                context.MessageFormatter.AppendArgument("Message", "Quota frame total target");
```

Comment:
The use of message argument her defeats the purpose of the WithMessage call above, i would create two separate validations for this :)